src/edit_collector.h

```diff
@@ -196,6 +208,31 @@ class EditCollector {
       return Status::OK();
     }
 
+    void Dump(bool with_keys) const {
+      std::vector<uint64_t> files;
+      files.reserve(added_files_.size());
+      for (auto& file : added_files_) {
+        files.push_back(file.first);
+      }
+      sort(files.begin(), files.end());
```

Comment:
I didn't see `using std::sort`, where is the leakage?

Reply:
updated